server/utils/serviceApp.js

```diff
@@ -9,6 +9,6 @@ var router = express.Router();
 
 router.use(express.static('./client'));
 router.use(express.static('./.tmp'));
-router.use(express.static('./bower_components'));
+router.use(express.static('./node_modules'));
```

Comment:
@himdel acceptable for me to submit a second pr making the above change? or is this a thing fun enough to rope @simaishi into?  👐 🙌